Assignment4/ddavidlorenzo-21A274/task06.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1RkaMx6480F8tC9f54HPbIwa1CBKE6Cty
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This won't run in Python. You need to comment it out (line 12). The same applies to the next task